### PR TITLE
Restructuring of build process

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,11 @@
-FROM alpine:3.4
+FROM plugins/base:multiarch
+MAINTAINER Drone.IO Community <drone-dev@googlegroups.com>
 
-RUN apk update && \
-  apk add \
-    ca-certificates && \
-  rm -rf /var/cache/apk/*
+LABEL org.label-schema.version=latest
```
Comment (tonglil) [inline Dockerfile]: I've been thinking about this, does this make sense/what does it mean when the version says it is latest?

**Comment by tonglil** [comment]
Another thought I had was you can restructure the build to be like this since Drone no longer enforces build/publish ordering:
```
build_linux_amd64 -> /release/drone-github-release
publish_linux_amd64
build_linux_arm -> /release/drone-github-release
publish_linux_arm
...
```
And then you can just symlink the 3 different dockerfiles from `Dockerfile`.

Not a significant change, but reduces surface area to update.

**Comment by tboerger** [comment]
A non-unique name will kill the parallel binary build and will increase the build time, so I'm not a big fan of this change

File: Dockerfile (hunk 1 above)
Comment (tboerger) [reply Dockerfile]: These labels will be shown on pages like https://microbadger.com/images/plugins/slac. I thought about displaying the real version there, but than it's a third place that needs to be updated which will be forgotten for sure

**Comment by tonglil** [comment]
Ah missed the builds were happening in parallel.

**Pull request merged**